<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin_panel</class>
 <widget class="QWidget" name="Admin_panel">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Панель администратора</string>
  </property>
  <widget class="QTableView" name="tableView_data">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>620</width>
     <height>190</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_search">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Поиск...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_new_admin_login">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>150</width>
     <height>18</height>
    </rect>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Введите Login</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_edit">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Редактировать</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_delete">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_add">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit_new_admin_password">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>150</width>
     <height>18</height>
    </rect>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Введите Пароль</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
